rest/codegen/paths_test.go

```diff
@@ -50,15 +50,125 @@ func ShowAccountPath3(id int32, view string) string {
 }
 
 `
+
+		pathWithStringSliceParam = `
+// ShowAccountPath returns the URL path to the Account service Show HTTP endpoint.
+func ShowAccountPath(slice_string []string) string {
+	encodedslice_string := make([]string, len(slice_string))
+	for i, v := range slice_string {
+		encodedslice_string[i] = url.QueryEscape(v)
```

Comment:
I really dislike the variable name generation for the arguments names.
Also its open to create invalid variable names. For example: a path param with the name `slice-string` would break at compile time.
This should be translated into `sliceString` and `encodedSliceString`

Probably we would need to introduce the function `Goify` here to create safe variable names.
What are your thoughts?